src/libs/ReportUtils.ts

```diff
@@ -7089,8 +7089,8 @@ function shouldShowMerchantColumn(transactions: Transaction[]) {
  */
 function isChatUsedForOnboarding(optionOrReport: OnyxEntry<Report> | OptionData): boolean {
     // onboarding can be an array for old accounts and accounts created from olddot
-    if (!Array.isArray(onboarding) && onboarding?.chatReportID === optionOrReport?.reportID) {
-        return true;
+    if (onboarding && !Array.isArray(onboarding) && onboarding.chatReportID && optionOrReport?.chatReportID) {
+        return onboarding.chatReportID === optionOrReport.chatReportID;
```

Comment:
We should check against optionOrReport reportID and not optionOrReport chatReportID. The latter seems like dupe information and is usually empty.
```suggestion
        return onboarding.chatReportID === optionOrReport?.reportID;
```